spec/lib/twingly/url_spec.rb

```diff
@@ -70,12 +70,30 @@ def valid_urls
 end
 
 def leading_and_trailing_whitespace
```

Comment:
A future improvement could be to generate all permutations instead of manually listing them. It looks like a tedious task over time

Reply:
Yeah for sure